packages/@aws-cdk/aws-events/lib/target.ts

```diff
@@ -65,4 +66,12 @@ export interface RuleTargetConfig {
    * @default the entire event
    */
   readonly input?: RuleTargetInput;
+
+  /**
+   * The construct that is backing this target.
```

Comment:
"construct" => "resource"

I realized recently that saying "construct" is like saying "class". It's a primitive from the programming model, not from the infrastructure space